web/profiles/joinup/joinup.profile

```diff
@@ -198,6 +198,16 @@ function joinup_inline_entity_form_reference_form_alter(&$reference_form, &$form
 }
 
 /**
+ * Implements hook_form_alter().
+ *
+ * Disable access to the revision information vertical tab.
+ * This prevents access to the revision log and the revision checkbox too.
+ */
+function joinup_workflows_form_node_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $form['revision_information']['#access'] = FALSE;
```

Comment:
I was looking in adding a test for this and it seems this code is never executed. This appears to be an implementation of `hook_form_FORM_ID_alter()` but there is no module named `joinup_workflows` so this function name is never considered as a valid hook implementation.

I have checked in the interface and the revision information is still visible when e.g. creating a news article.

So yeah let's add a test, and fix it too :)